cmd/dapr.go

```diff
@@ -43,26 +42,22 @@ Distributed Application Runtime`,
 
 var logAsJSON bool
 
+var cliVersion string
+var outputInJSON string
```

Comment:
If we don't plan to support output as yaml, then this flag should be 'bool'

Reply:
Although we don't plan to support yaml output，but it's strange if change --output to bool. I prefer as --output json.